streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -17,40 +17,34 @@
 
 package org.apache.kafka.streams.processor.internals.assignment;
 
+
 import java.util.HashSet;
 import java.util.Set;
 
 public class ClientState<T> {
```

Comment:
Not part of this PR: but why does `ClientState` have a generic type? It's only used with `TaskId` anyway? Or did I miss anything?